icons/shields-dot-io.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Shields.io icon</title><path d="M.677 8.515l-.145.071c-.185.09-.37.276-.46.461L0 9.192v5.587l.065.138c.128.273.38.476.673.542.104.023 2.55.028 11.318.025l11.185-.004.126-.052c.27-.11.462-.303.573-.574.05-.123.05-.147.058-2.772.005-1.915 0-2.688-.02-2.792a1 1 0 00-.55-.71l-.139-.065H11.983zm8.54.907h7.773l-.006 2.281-.007 2.281-.073.143a.726.726 0 01-.397.346c-.067.02-2.087.028-7.59.028H1.42l-.153-.08a.689.689 0 01-.359-.462c-.017-.086-.024-.747-.019-2.071l.007-1.946.068-.139a.743.743 0 01.171-.223c.21-.172-.52-.158 8.084-.158zm11.728 1.224c.48.005.964.25 1.169.697.275.571.23 1.326-.214 1.802-.554.6-1.678.483-2.075-.238a1.583 1.583 0 01-.205-.78c-.033-.58.284-1.212.857-1.404.149-.053.308-.078.468-.077zm-2.228.053h.368v2.783h-.368V11.72zm2.28.266c-.387-.012-.786.205-.915.584a1.866 1.866 0 00-.082.583c-.046.52.351 1.075.896 1.088.46.027.918-.323.971-.79.097-.453.017-1.007-.377-1.299a.87.87 0 00-.493-.166zM2.871 12.54l.348-.03c.013.255.183.502.434.573.296.109.651.097.919-.08.18-.137.226-.446.039-.598-.19-.16-.448-.183-.677-.25-.285-.068-.597-.137-.804-.362a.688.688 0 01.036-.904c.248-.257.634-.31.974-.276.299.03.615.163.76.441.051.11.133.273.08.378-.108.021-.22.018-.33.03-.01-.253-.21-.495-.47-.515-.23-.038-.488-.03-.691.098-.17.12-.225.408-.046.546.155.122.36.145.544.2.323.075.68.135.923.383.243.266.23.71-.01.973-.245.282-.645.36-1.001.33-.314-.018-.647-.125-.84-.387a.932.932 0 01-.188-.55m2.679.894v-2.783h.341v.999a.789.789 0 01.712-.272c.24.019.49.167.548.413.062.24.034.489.04.734v.91H6.85c-.001-.447.002-.894-.002-1.34-.003-.164-.076-.345-.243-.4-.266-.097-.617.048-.681.338-.05.214-.026.435-.032.653v.748H5.55m2.165-2.39v-.393h.342v.393zm0 2.39v-2.016h.342v2.016zm2.244-.65l.353.044c-.085.421-.528.701-.945.655-.393-.008-.781-.283-.858-.68-.102-.402-.049-.885.255-1.189.375-.364 1.057-.318 1.365.108.167.225.204.519.192.791H8.818c-.018.351.275.73.649.688a.56.56 0 00.491-.416m-1.121-.553h1.125c.016-.337-.318-.658-.658-.572a.586.586 0 00-.467.572m1.898 1.202v-2.783h.342v2.783zm2.18 0v-.254c-.17.296-.58.372-.878.236-.289-.124-.482-.413-.528-.72-.061-.335-.015-.7.169-.991.226-.329.725-.443 1.058-.212.051.028.125.14.157.124v-.966h.34v2.783h-.317m-1.08-1.006c-.01.273.077.59.339.723a.497.497 0 00.666-.26c.11-.242.091-.519.048-.775-.052-.236-.249-.462-.506-.462a.514.514 0 00-.498.387 1.33 1.33 0 00-.05.387m1.801.405l.338-.054c.015.206.184.395.393.41.17.025.366.014.5-.104.099-.091.135-.273.02-.366-.11-.088-.257-.098-.386-.142-.219-.06-.451-.103-.637-.24a.528.528 0 01-.012-.764c.191-.183.476-.217.729-.194.204.022.43.091.55.27a.75.75 0 01.103.29l-.334.045c-.015-.19-.2-.336-.388-.328-.15-.01-.328.002-.438.12-.076.088-.066.24.038.301.122.086.276.092.412.143.213.062.443.103.625.238.204.18.21.516.048.725-.158.217-.437.294-.694.297a1.15 1.15 0 01-.522-.108.738.738 0 01-.345-.54"/></svg>
```

Comment:
Hi @klu17, thanks for updating this!
Just having a quick look, it appears there's still a small difference (see below.)

That may just be down to SVGO, so could you try with 4 decimals please? 🙏🏻

![image](https://user-images.githubusercontent.com/1830867/103234184-eab93500-4936-11eb-9da8-1b19eb8d2121.png)